flatgfa/src/packedseq.rs

```diff
@@ -312,8 +419,62 @@ fn test_getter_setter() {
         Nucleotide::C,
         Nucleotide::C,
     ]);
-    assert_eq!(vec.get(0), Nucleotide::A);
-    assert_eq!(vec.get(1), Nucleotide::A);
+    assert_eq!(vec.as_ref().get(0), Nucleotide::A);
+    assert_eq!(vec.as_ref().get(1), Nucleotide::A);
     vec.set(1, Nucleotide::G);
-    assert_eq!(vec.get(1), Nucleotide::G);
+    assert_eq!(vec.as_ref().get(1), Nucleotide::G);
+}
+
+#[test]
+fn test_export_import_simple() {
+    let vec = PackedSeqStore::create(vec![
+        Nucleotide::A,
+        Nucleotide::C,
+        Nucleotide::T,
+        Nucleotide::G,
+    ]);
+    let input = vec.as_ref();
+    let filename = "capra_test_file";
+    let num_bytes = total_bytes(&input);
+    let mut mem = map_new_file(filename, num_bytes as u64);
+    let mut buf = vec![0u8; num_bytes];
+    let buf_ref = &mut buf;
+    input.dump(buf_ref);
+    mem[..buf_ref.len()].copy_from_slice(buf_ref);
+    let result: &[u8] = &mem;
+    let output = PackedSeqView::view(result);
+    assert_eq!(input.get(0), output.get(0));
+    assert_eq!(input.get(1), output.get(1));
+    assert_eq!(input.get(2), output.get(2));
+    assert_eq!(input.get(3), output.get(3));
+    std::mem::drop(mem);
+    let _ = std::fs::remove_file(filename);
+}
+
+#[test]
+fn test_export_import() {
+    let len = 10;
+    let num_trials = 10;
+    let mut rng = rand::thread_rng();
+    for _ in 0..num_trials {
+        let mut vec: Vec<Nucleotide> = Vec::new();
+        for _ in 0..len {
+            let rand_num = rng.gen_range(0..=3);
+            match rand_num {
+                0 => vec.push(Nucleotide::A),
+                1 => vec.push(Nucleotide::C),
+                2 => vec.push(Nucleotide::T),
+                3 => vec.push(Nucleotide::G),
+                _ => panic!("Incorrect item appended!"),
+            }
+        }
+        let old_vec = vec.clone();
```

Comment:
`vec` itself is never mutated or destroyed, so do we need to clone it here? Can we just reuse `vec` itself instead of creating `old_vec`?